normative_content.rst

```diff
@@ -0,0 +1,29 @@
+IATI Normative and Non-normative content
+========================================
+
+**Normative:** `Normative content <https://en.wikipedia.org/wiki/Normative#Standards_documents>`__ is the prescriptive part of a standard. It sets the rules to be followed in order to be evaluated as compliant with the standard, and from which no deviation is permitted. 
+
+The following content is considered normative:
+
+- `IATI SSOT <https://github.com/IATI/IATI-Standard-SSOT>`__
+
+  - `IATI-Codelists <https://github.com/IATI/IATI-Codelists/tree/7dc8ae401dfdf6970130eeabc08974b11ff9140b>`__
```

Comment:
Did you intend to include the /tree/{commit_sha} here? If you just want to link to the general IATI-Codelists repo we should shorten the URL. Adding the /tree/{commit_sha} will lock the link into that specific commit in history.
```suggestion
  - `IATI-Codelists <https://github.com/IATI/IATI-Codelists>`__
```

Reply:
Good spot! I'd just copied and pasted from the google doc. I'll change them